deployments/cluster.nix

```diff
@@ -28,23 +38,16 @@
     };
 
     services.sshguard.enable = true;
+
+    networking.firewall.allowedTCPPorts = [ 22 ];
   };
 
-  witness-1 = import ./cluster/witness.nix 1;
-  witness-2 = import ./cluster/witness.nix 2;
-  witness-3 = import ./cluster/witness.nix 3;
-  witness-4 = import ./cluster/witness.nix 4;
+  resources = import ./cluster-resources.nix { inherit region env domain; };
 
   witness-load-balancer = import ./cluster/witness-load-balancer.nix domain;
 
-  resources.ec2KeyPairs.cluster-key = { inherit region; };
-
-  resources.ec2SecurityGroups.cluster-ssh = {
-    inherit region;
-    rules = [{
-      fromPort = 22;
-      toPort = 22;
-      sourceIp = "0.0.0.0/0";
-    }];
-  };
+  witness0 = import ./cluster/witness.nix 0;
+  witness1 = import ./cluster/witness.nix 1;
+  witness2 = import ./cluster/witness.nix 2;
+  witness3 = import ./cluster/witness.nix 3;
```

Comment:
Why starting from zero? Upstream counts committee participants starting from 1.

Reply:
Zero is the private hidden node. Public nodes start at 1. You'll notice node 0 is also not in the load balancer array for nginx.

Reply:
By the way, committee participants are indexed base-0 (had to fix that last night, last witness refused to start because index out of bounds)